tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -36,20 +36,6 @@ struct choose_literal<wchar_t> {
 

```

Comment:
This test should no longer be guarded for `defined(__cpp_lib_concepts)` because this is guaranteed by the `concepts_matrix`. (Apologies for not clearly explaining this in my previous comment.)

Reply:
I could have inferred this, my brain just didn't cooperate 😬